### PR TITLE
Prune launcher binaries for other archs

### PR DESCRIPTION
**Pull request by wendigo**
## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
## Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by nineinchnick — APPROVED** [review]

File: core/docker/Dockerfile [hunk 1]
```diff
@@ -49,6 +50,9 @@ COPY --chown=trino:trino trino-server-${TRINO_VERSION} /usr/lib/trino
 COPY --chown=trino:trino default/etc /etc/trino
 COPY --chown=trino:trino --from=jvmkill /libjvmkill.so /usr/lib/trino/bin
 
+RUN rm -Rf $(ls -d /usr/lib/trino/bin/linux-* | grep -v "linux-${ARCH}") && \
```
[the remark this answers is not in the document]
Comment (wendigo) [reply core/docker/Dockerfile]: Done.

File: core/docker/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# syntax=docker/dockerfile:1-labs
```
Comment (losipiuk) [inline core/docker/Dockerfile:1]: nit: separate commit

File: core/docker/Dockerfile [hunk 3]
```diff
@@ -45,7 +47,8 @@ RUN \
 
 ARG TRINO_VERSION
 COPY --chown=trino:trino trino-cli-${TRINO_VERSION}-executable.jar /usr/bin/trino
-COPY --chown=trino:trino trino-server-${TRINO_VERSION} /usr/lib/trino
+COPY --chown=trino:trino --exclude=bin/darwin-* --exclude=bin/linux-* trino-server-${TRINO_VERSION} /usr/lib/trino
+COPY --chown=trino:trino trino-server-${TRINO_VERSION}/bin/linux-${ARCH} /usr/lib/trino/bin/linux-${ARCH}
```
Comment (losipiuk) [inline core/docker/Dockerfile:51]: do we have just `/bin/linux-*`` in `trino-server-${TRINO_VERSION}`?

**Review by losipiuk — APPROVED** [review]

File: core/docker/Dockerfile (hunk 3 above)
Comment (losipiuk) [reply core/docker/Dockerfile:51]: Oh nvm - you copy the rest above. LGTM

File: core/docker/Dockerfile (hunk 2 above)
Comment (wendigo) [reply core/docker/Dockerfile:1]: No, it's part of that one, --exclude requires it

Comment (losipiuk) [reply core/docker/Dockerfile:1]: really... Looks like comment for vim syntax coloring.
Awful.

**Pull request merged**